uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -826,6 +829,27 @@ define([
       vm.dismissModal();
     }
 
+    /**
+     * Stores the request from and to times original values
+     */
+    function rememberRequestOriginalTimes () {
```

Comment:
I dont think we need this new variable and function. We already have `initialLeaveRequestAttributes` which stores the request object's original state. So we can use the same.